how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -71,23 +78,29 @@ EOT
 
 ### Configure Kubelet to use containerd
 
-In order to allow kubelet use containerd (using CRI interface) configure the service to
-point to containerd socket.
+In order to allow kubelet use containerd (using CRI interface), configure the service to
+point to the `containerd` socket.
+
+
+- Configure k8s to use containerd
 
 ```bash
-# Configure k8s to use containerd
 $ sudo mkdir -p  /etc/systemd/system/kubelet.service.d/
+```
+```bash
 $ cat << EOF | sudo tee  /etc/systemd/system/kubelet.service.d/0-containerd.conf
 [Service]                                                 
 Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --runtime-request-timeout=15m --container-runtime-endpoint=unix:///run/containerd/containerd.sock"
 EOF
+```
+```bash
 $ sudo systemctl daemon-reload
 ```
 
 ### Optional: Configure proxy
 
-If you are behind a proxy this script will configure your proxy for docker
-kubelet and containerd.
+If you are behind a proxy, use this script to configure your proxy for docker,
```

Comment:
If you are behind a proxy, use the following script to configure your proxy for docker,